### PR TITLE
Update index.html - typo corrections, minimal fine-tuning of phrasing

### PR DESCRIPTION
**Pull request by o4tuna**

### CODE REVIEW

**Comment by gr2m** [comment]
This looks great, thanks @o4tuna for the improvements! I’m sorry for the delay, must have slipped through our reviews :( I’ll have someone else have a double check then we can merge

File: contribute/index.html [hunk 1]
```diff
@@ -12,10 +12,10 @@
...
         </p>
         <p>
-            If you want to take a first look at <strong>what we've been working recently</strong>, a good start may be to look at our <a href="http://gr2m.github.io/milestones/" target="_blank">Milestones</a>, the <a href="http://espy.github.io/ubersicht/#hoodiehq" target="_blank">Ubersicht</a> of our activity on Hoodie's <a href="http://github.com/hoodiehq" target="_blank">GitHub</a> activity, — and of course our <a href="/blog">Blog</a> and our <a href="http://twitter.com/hoodiehq" target="_blank">Twitter</a> always offer the latest information as well.
+            If you want to take a first look at <strong>what we've been working recently</strong>, a good start may be to look at our <a href="http://gr2m.github.io/milestones/" target="_blank">Milestones</a>, the <a href="http://espy.github.io/ubersicht/#hoodiehq" target="_blank">Ubersicht</a> of our activity on Hoodie's <a href="http://github.com/hoodiehq" target="_blank">GitHub</a> activity, — and of course our <a href="/blog">blog</a> and our <a href="http://twitter.com/hoodiehq" target="_blank">Twitter</a> always offer the latest information as well.
```
Comment (varjmes) [inline contribute/index.html]: Could we change "what we've been working recently" to "what we've been working on recently"?

**Comment by varjmes** [comment]
Hi @o4tuna, noticed a typo I thought you might like to help us fix and commented on it.
Let us know if you need help fixing it.
Thanks so much for your PR!

File: contribute/index.html (hunk 1 above)
Comment (o4tuna) [inline contribute/index.html]: You're right--"on" is missing from the sentence. If you still see the version missing the word "on" after you get this comment, it means that I haven't figured out how to edit it so please feel free to make the change if you can.

Comment (o4tuna) [inline contribute/index.html]: Okay, I think I figured out how to make the correction. The sentence should now include the missing word "on". Thank you, Charlotteis!

**Comment by gr2m** [comment]
LGTM now, thanks :)

**Comment by janl** [comment]
lgtm

**Comment by gr2m** [comment]
merged via 296d5c9, thanks :+1:

**Pull request closed without merging**